recipes-app/iot2050-eio-manager/README.md

```diff
@@ -0,0 +1,24 @@
+# IOT2050 Extended IO Subsystem
```

Comment:
Commit message still says "Complete the commit message" :wink:

Reply:
Now it should be a reviewable states.